nl_server/config.py

```diff
@@ -73,11 +82,15 @@ def load(embeddings_map: Dict[str, Dict[str, str]]) -> List[EmbeddingsIndex]:
   #
   # Download all the models.
   #
-  models_set = set([i.model_name for i in indexes if i.model_name])
+  models_set = set()
+  for i in indexes:
+    if i.model_name and i.model_type == ModelType.LOCAL:
+      models_set.add(i.model_name)
```

Comment:
Can we assert or fail if for VERTEXAI model_name isn't in the config[VERTEX_AI_MODELS] map?

Reply:
hm, I tried accessing current_app.config from this point, but I don't think the app config is set yet at this point. I could try passing the config through, but that also feels kind of messy. Maybe it could be better to just have a test to check that embeddings.yaml refers to valid vertex ai models?